debian/gen-fixtures.sh

```diff
@@ -51,5 +51,6 @@ reprepro -C jotunheimr includedeb ragnarok jotunheimr/*.deb
 
 rm dists/ragnarok/jotunheimr/binary-armeb/Packages
 
-cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/dists "${OUTPUTDIR}"
-cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/pool "${OUTPUTDIR}"
+mkdir -p "${OUTPUTDIR}"
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/dists -t "${OUTPUTDIR}"
+cp -r --no-preserve=mode --reflink=auto "${TMPDIR}"/pool -t "${OUTPUTDIR}"
```

Comment:
Do we need both the `-t` and the `mkdir -p`?

Reply:
No, the `-t` would fail if the directory didn't exist. So it is there as a safeguard.